### PR TITLE
Add desktop version portfolio

### PR DESCRIPTION
**Pull request by ONJoseph**
-Added desktop images 
-Modified index.html file
-Modified style.css file
-Updated navbar
-Added animation and transition
-Added media queries

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -49,7 +49,7 @@
         <section class="head flex">
           <h1 class="head__title fz-40 fw-700 lh-52">I'm Joseph Glad to see you!</h1>
```
Comment (Kingstalux) [inline index.html]: - [x] Great job so far 👍 Kindly make this heading to take up two lines just like in the template 
- Your design:
Annotation 2022-03-24 215822
- Figma design:
Annotation 2022-03-24 215834

**Review by Kingstalux — CHANGES_REQUESTED** [review]
Hi @ONJoseph, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✅ Great responsive design 🔥 
✅ Linters are passing
✅ Good pull request description

## Required Changes ♻️
_Check the comments under the review._
- [x] Kindly increase that space between the technology categories (languages, frameworks, and skills) and their `arrows` so as to follow the Figma design.
- Your design:
Annotation 2022-03-24 220007
- Figma design:
Annotation 2022-03-24 220019



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ONJoseph** [comment]
Thanks a lot for the review and suggested changes👍👍
Will implement ASAP.

File: index.html (hunk 1 above)
Comment (ONJoseph) [reply index.html]: Done. Thanks a lot. @Kingstalux

File: styles/style.css [hunk 2]
```diff
@@ -445,3 +445,194 @@ p {
...
+    padding: 6rem;
+    flex-direction: row;
+    -moz-column-gap: 37px;
+    column-gap: 37px;
+    border-top-right-radius: 8rem;
+  }
+
+  .about__myself {
+    flex-basis: 25rem;
+  }
```
Comment (phelian23) [inline styles/style.css]: - [x] As requested by the previous reviewer, Kindly fix the margin of the technology categories (languages, frameworks, and skills) and their arrows so as to follow the Figma design.
your design
Capture

**Review by phelian23 — CHANGES_REQUESTED** [review]
Hi @ONJoseph, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles/style.css (hunk 2 above)
Comment (ONJoseph) [reply styles/style.css]: I have implemented the suggestions. @Kingstalux  Please can you give me some feedback before I resubmit? As this is my last chance. Thanks a lot.

Comment (ONJoseph) [reply styles/style.css]: > * [x]  As requested by the previous reviewer, Kindly fix the margin of the technology categories (languages, frameworks, and skills) and their arrows so as to follow the Figma design.
>       your design
>       Capture

Suggested change implemented. Thanks a lot.

Comment (Kingstalux) [reply styles/style.css]: Hello @ONJoseph, 
The arrows are okay now but you've removed the white background from the about section. Kindly add back the white background and you'll be good to go.
Annotation 2022-03-25 052514

Comment (ONJoseph) [reply styles/style.css]: I have implemented the suggestion. @Kingstalux Please is it good to go now? As this is my last chance. Thanks a lot.

Comment (Kingstalux) [reply styles/style.css]: Hello @ONJoseph ,
- The white background is supposed to extend right up to the contact section and the top right of the about section should be curved.
- Also, there is to be some space between the top of the about section and the about heading. 
- Your design:
Annotation 2022-03-25 201500
- Figma design:
Annotation 2022-03-25 201441

Comment (ONJoseph) [reply styles/style.css]: Grateful for the kind feedback @Kingstalux . I have tried to effect the change requested. Please can I proceed to submit for final approval? As this is my last submission chance. Thanks.

Comment (Kingstalux) [reply styles/style.css]: yeah, you can submit now for approval 👍

File: styles/styles.scss [hunk 3]
```diff
@@ -472,4 +472,198 @@ p {
       }
     }
   }
-}
\ No newline at end of file
+}
```
Comment (rustam-tolipov) [inline styles/styles.scss:475]: - [x] [OPTIONAL] **GOOD JOB GUYS ⭐🙌👏**. I see you struggled with the spacing and background bug in the about section but don't worry I am here to help you 😄.

1. First remove the `margin-top: 150px;` from contact section in media query 992
2. Add this `padding: 9.6rem 3.2rem;` to `.about_bg-white_flex` inside 992px media query
It should already fix the problem but there are some issues in the mobile version also for this do these steps
3. Add this to the about section in CSS just to the mobile version, code, not inside a media query

```css
    background: white;
    padding: 6.4rem 2.4rem;
```
All of these worked on my browser 😀👌

**Review by rustam-tolipov — APPROVED** [review]
## Hello team👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉, 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by ONJoseph** [comment]
Thanks immensely for the approval and kind detailed ***@***.***
Will keep improving the project.

File: styles/styles.scss (hunk 3 above)
Comment (ONJoseph) [reply styles/styles.scss:475]: Changes effected. Looks good! Thanks a lot!@Rustamxon7

**Comment by ONJoseph** [comment]
Many thanks 👍👍

**Pull request merged**